Tests/LibWeb/Text/input/SVG/svg-pattern-element-attributes.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<script src="../include.js"></script>
+<svg xmlns="http://www.w3.org/2000/svg" style="display: none">
+    <pattern id="pattern-element" x="10" y="20" width="30" height="40"></pattern>
+</svg>
+<script>
+    test(() => {
+        const patternElement = document.getElementById("pattern-element");
+
+        println("Element found: " + (patternElement !== null));
+        println("id: " + patternElement.getAttribute("id"));
+        println("x: " + patternElement.getAttribute("x"));
+        println("y: " + patternElement.getAttribute("y"));
+        println("width: " + patternElement.getAttribute("width"));
+        println("height: " + patternElement.getAttribute("height"));
+    });
```

Comment:
I think I misspoke on this. To test the actual bindings, these need to use what's defined in the .idl file, eg `patternElement.x` or `patternElement.patternUnits`. But at the moment the ones you've defined just return 0 so that wouldn't be a very interesting test. :sweat_smile:

If you're interested in doing a bit more and making this more useful, take a look at `SVGFilterElement.cpp`. In `SVGFilterElement::apply_presentational_hints()` it applies the x/y/width/height attributes as CSS properties, and then the getters for those use `svg_animated_length_for_property()`.